backend/domain/domainmodels.js

```diff
@@ -0,0 +1,134 @@
+
+
+
+
+function Room (){
+
+        this.roomName = this.setRoomName(); //@@@@@@@ -some function to initialise a roomID
+
+        this.gamestate = GameStateEnum["Day Time"];
+
+        this.maxPlayerCount = 6;
+
+        this.players = new Array(maxPlayerCount);
+
+        this.roundNumber = 0;
+
+        this.getRoomName = function () {
+            return this.roomName;
+        };
+
+        this.getGamestate = function () {
+            return this.gamestate;
+        };
+
+        this.setGamestate = function (gamestate) {
+            this.gamestate = gamestate;
+        };
+
+
+        this.getPlayers = function () {
+            return this.players;
+        };
+
+
```

Comment:
Sorry to be picky, but the spacing between functions is inconsistent. Maybe we can keep it at 1 empty line?

Reply:
I have fixed the white space inconsistencies